packages/theming/atlas/src/themesource/atlas_ui_resources/native/core/helpers/images.ts

```diff
@@ -28,80 +28,51 @@ export const imageOverlay = {
     }
 };
 //
-// Image Sizes
+// Image Shapes
 export const imageSquare = {
     image: {
-        width: "100%",
-        height: "auto",
-        aspectRatio: 1,
-        borderRadius: border.radiusSmall
-    }
-};
-export const imageSquareSmall = {
-    image: {
-        aspectRatio: 1,
-        width: image.image.small,
-        height: image.image.small,
         borderRadius: border.radiusSmall
     }
 };
-export const imageSquareMedium = {
+export const imageCircle = {
     image: {
-        aspectRatio: 1,
-        width: image.image.medium,
-        height: image.image.medium,
-        borderRadius: border.radiusSmall
+        borderRadius: 50000
     }
 };
-export const imageSquareLarge = {
-    image: {
-        aspectRatio: 1,
-        width: image.image.large,
-        height: image.image.large,
-        borderRadius: border.radiusSmall
-    }
-};
-export const imageSquareLarger = {
-    image: {
-        aspectRatio: 1,
-        width: image.image.larger,
-        height: image.image.larger,
-        borderRadius: border.radiusSmall
-    }
-};
-export const imageCircleSmall = {
+// Image Sizes
+export const imageSmall = {
     image: {
         width: image.image.small,
         height: image.image.small,
-        borderRadius: image.image.small / 2
+        aspectRatio: 1
```

Comment:
aspect ratio??

Reply:
+ if you are using aspectRatio 1 you dont need to specify height

Reply:
Fixed